apps/settings/js/modules/dialog/alert_dialog.js

```diff
@@ -21,6 +22,15 @@ define(function(require) {
   };
 
   return function ctor_alertDialog(panelDOM, options) {
-    return new AlertDialog(panelDOM, options);
+    var d = new AlertDialog(panelDOM, options);
```

Comment:
please use more meaningful variable name, such as `dialog` if the name is not conflicted